src/src/EmailBundle/Repository/EmailLogRepository.php

```diff
@@ -10,4 +12,28 @@
  */
 class EmailLogRepository extends \Doctrine\ORM\EntityRepository
 {
+    public function isEmailSent($templateName, $email, $emailType, $eventType, $event)
+    {
+        $qb = $this->_em->createQueryBuilder('el');
+        $qb->select('count(el.id)')
+            ->from(EmailLog::class, 'el')
+            ->where('el.emailAddress LIKE :email')
+            ->andWhere('el.emailType LIKE :type')
+            ->andWhere('el.eventType = :eventType')
+            ->andWhere('el.template LIKE :template')
+            ->andWhere('el.status = 0')
+            ->setParameters(array(
+                'email' => $email,
+                'type' => $emailType,
```

Comment:
dal by som `'emailType'` takto som si myslel na prvy pohlad, ze je to duplicitne s `$eventType`